waspc/cli/src/Wasp/Cli/Command/CreateNewProject.hs

```diff
@@ -103,5 +127,71 @@ writeMainWaspFile waspProjectDir (ProjectInfo projectName appName) = IOUtil.writ
           "}"
         ]
 
+createProjectFromTemplate :: Path System Abs (Dir WaspProjectDir) -> ProjectInfo -> Command ()
+createProjectFromTemplate absWaspProjectDir ProjectInfo {_appName = appName, _projectName = projectName, _template = maybeTemplateName} = do
```

Comment:
Interesting that this compiles, I would expect Haskell to require  ( ) around `ProjectInfo { ...}`.